build.gradle.kts

```diff
@@ -33,6 +33,42 @@ oci {
             optionalCredentials()
         }
     }
+    imageMapping {
+        mapModule("com.hivemq", "hivemq-enterprise") {
+            toImage("hivemq/hivemq4")
+        }
+    }
+    imageDefinitions {
+        register("main") {
+            allPlatforms {
+                dependencies {
+                    runtime("com.hivemq:hivemq-enterprise:latest") { isChanging = true }
+                }
+                layers {
+                    layer("hivemqExtension") {
+                        contents {
+                            permissions("opt/hivemq/", 0b111_111_101)
+                            permissions("opt/hivemq/extensions/", 0b111_111_101)
+                            into("opt/hivemq/extensions") {
+                                permissions("*/", 0b111_111_101)
+                                permissions("*/dnsdiscovery.properties", 0b110_110_100)
+                                permissions("*/hivemq-extension.xml", 0b110_110_100)
+                                from(zipTree(tasks.hivemqExtensionZip.flatMap { it.archiveFile }))
+                            }
+                        }
+                    }
+                }
+            }
+        }
+        register("integrationTest") {
+            allPlatforms {
+                dependencies {
+                    runtime(project)
+                    runtime("com.hivemq.extensions:hivemq-prometheus-extension")
```

Comment:
I probably have a wrong assumption but I was curious: do we need a mapping for the prometheus-extension?

Reply:
The include is optional, because it is only required when executing the integration tests, not for other tasks.